provider/pihole/clientV6.go

```diff
@@ -63,6 +64,7 @@ func newPiholeClientV6(cfg PiholeConfig) (piholeAPI, error) {
 			},
 		},
 	}
+
 	cl := instrumented_http.NewClient(httpClient, &instrumented_http.Callbacks{})
 
 	p := &piholeClientV6{
```

Comment:
Should we remove the instrumented http client, as mentioned [here](https://github.com/kubernetes-sigs/external-dns/pull/5226#issuecomment-2781117293)?

Reply:
https://github.com/kubernetes-sigs/external-dns/pull/5253#issuecomment-2781536142